package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
```

Comment:
## Your Details Page
![image](https://user-images.githubusercontent.com/93039359/225508635-f5571138-e6c0-46fe-a334-93203f13bbbd.png)

## Design Details Page
![image](https://user-images.githubusercontent.com/93039359/225508858-e4bd2d09-bb9e-405a-b795-36227f349f01.png)

**[OPTIONAL]**

- [ ] Great implementation of the design but I guess the design you have implemented for your details page was actually for your home page as you can see in the second picture above how your details page should look like 😊
- [ ] It would be nice if you add those missing things I have highlighted with a blue marker in the Design image above 👍

Kindly make both of these changes to make your project look professional and follow all the design requirements 😊